### PR TITLE
LoadDetailsScreen

### PR DESCRIPTION
**Pull request by Shikhar-Pandey**

### CODE REVIEW

File: lib/detailCard.dart [hunk 1]
```diff
@@ -14,98 +15,159 @@ class DetailCard extends StatelessWidget {
...
-                                width: 8,
-                              ),
-                              Expanded(
-                                child: Text(
-                                  "$loadingPoint",
-                                  style: TextStyle(
-                                      fontSize: size_9,
-                                      color: unloadingPointTextColor,
-                                      fontWeight: mediumBoldWeight),
+      GestureDetector(onTap: (){
```
Comment (liveasy97) [inline lib/detailCard.dart:41]: can you use getx instead of Navigator? you can pass all these details in arguments

Comment (Shikhar-Pandey) [reply lib/detailCard.dart:41]: ok, I'll try.

File: lib/detailCard.dart [hunk 2]
```diff
@@ -14,98 +15,159 @@ class DetailCard extends StatelessWidget {
...
-      {this.loadingPoint,
-      this.unloadingPoint,
-      this.productType,
-      this.truckPreference,
-      this.noOfTrucks,
-      this.weight,
-      this.isPending,
-      this.comments,
-      this.isCommentsEmpty});
+  DetailCard({
```
Comment (liveasy97) [inline lib/detailCard.dart:26]: forgot to mention earlier, change this name to somewhat more descriptive name as DetailCard will get confusing when we will have so many apis to work with. Try to include the name of api with which the widget is related like in your case loadApi. also try to do it other places wherever possible

Comment (Shikhar-Pandey) [reply lib/detailCard.dart:26]: okay

File: lib/widgets/bookNowButtonAlertDialog.dart [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                    ),
+                    SizedBox(
+                      height: space_2,
+                    ),
+                    Container(
+                      height: space_7+2,
+                      width: MediaQuery.of(context).size.width,
+                      decoration: BoxDecoration(
+                          borderRadius: BorderRadius.circular(22),
+                          border: Border.all(color: Color(0xFF878787))),
```
Comment (liveasy97) [inline lib/widgets/bookNowButtonAlertDialog.dart:37]: don't forget the constants file use those files

File: lib/widgets/bookNowButtonAlertDialog.dart [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+                        padding:  EdgeInsets.only(
+                          left: space_2-2,
+                          right: space_2-2,
+                        ),
+                        child: DropdownButtonHideUnderline(
+                          child: DropdownButton<String>(
+                            value: _dropdownvalue1,
+                            icon: Icon(Icons.arrow_drop_down_circle_sharp),
+                            iconEnabledColor: darkBlueColor,
+                            style: TextStyle(fontSize: size_7,fontWeight: regularWeight,color: Colors.black),
```
Comment (liveasy97) [inline lib/widgets/bookNowButtonAlertDialog.dart:48]: same here

**Review by liveasy97 — APPROVED** [review]

File: lib/widgets/bookNowButtonAlertDialog.dart (hunk 3 above)
Comment (Shikhar-Pandey) [reply lib/widgets/bookNowButtonAlertDialog.dart:37]: done

File: lib/widgets/bookNowButtonAlertDialog.dart (hunk 4 above)
Comment (Shikhar-Pandey) [reply lib/widgets/bookNowButtonAlertDialog.dart:48]: done

**Comment by liveasy97** [comment]
changes have been merged

**Pull request closed without merging**